src/View/Cell.php

```diff
@@ -181,12 +181,6 @@ public function render($template = null)
 
             $builder = $this->viewBuilder()->setLayout(false);
 
-            if ($template !== null &&
-                strpos($template, '/') === false &&
-                strpos($template, '.') === false
-            ) {
-                $template = Inflector::underscore($template);
```

Comment:
Do we have any tests for template with absolute or relative paths.

Reply:
Not for cells specifically but I believe we do for `View` class itself.

Reply:
Looks like we have `testCellRenderPluginTemplate` which was added in b1041c6b90c  for the `.` scenario.